src/main/java/com/beerjournal/breweriana/item/rating/ItemIdAndRatings.java

```diff
@@ -0,0 +1,15 @@
+package com.beerjournal.breweriana.item.rating;
+
+import com.beerjournal.breweriana.item.rating.persistence.Rating;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.bson.types.ObjectId;
+
+import java.util.Set;
+
+@RequiredArgsConstructor
+@Data
```

Comment:
In this case there is no very important reason for creating static factory method, but all our model classes have such methods. For me it makes class similar to Scala case class and I think constructors are old-fashioned :D